src/app/_services/api/base.service.ts

```diff
@@ -65,7 +70,11 @@ export class BaseService<T extends Resource> {
 
     return this.httpClient
       .post(`${this.helperService.getApiUrl()}/${endpoint}${paginationTmp}`, requestPayload)
-      .pipe(map((data: any) => this.convertData(data)));
+      .pipe(
+        map((data: any) => {
+          return withPagination ? this.convertData(data) : data;
```

Comment:
like Daniel said this needs to be corected, so without pagination can also be serialized. Please use the this.convertData method for both and decide in that method what to return if it is paginated or not

Reply:
Resolved.